app/project_type/git.py

```diff
@@ -158,7 +160,71 @@ def _fetch_project(self):
     def _execute_in_repo_and_raise_on_failure(self, command, message):
         self._execute_and_raise_on_failure(command, message, self._repo_directory)
 
-    def _execute_git_remote_command(self, command, cwd=None, timeout=10):
+    def execute_command_in_project(self, *args, **kwargs):
+        """
+        Execute a command inside the repo. See superclass for parameter documentation.
+        """
+        # There is a scenario where self.project_directory doesn't exist yet (when a certain repo has never been
+        # fetched before on this particular machine). In order to avoid having python barf during this scenario,
+        # we have only pass in the cwd if it exists.
```

Comment:
Oh, I meant to comment -- these two methods (`execute_command_in_project` and `timing_file_path`) have not changed at all. They just moved up in the file. The change here was moving the method `_execute_git_remote_command` into the new class `_GitRemoteCommandExecutor`.